pyproject.toml

```diff
@@ -9,6 +9,7 @@ build-backend = "mesonpy"
 requires = [
     "meson-python>=0.14.0",
     "Cython>=0.29.35",
+    "ninja>=1.10.0",
```

Comment:
This change is not desirable. `meson-python` will take care of installing `ninja` as necessary